setup.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+curl -fsSL https://get.docker.com | /bin/sh
+
+DOJO_PATH="./dojo"
+git clone https://github.com/HUSTSeclab/dojo.git "$DOJO_PATH"
+docker build -t pwncollege/dojo "$DOJO_PATH"
+docker run --privileged -d -v "${DOJO_PATH}:/opt/pwn.college:shared" -p 22:22 -p 80:80 -p 443:443 --name dojo pwncollege/dojo
```

Comment:
like before, please revise those port numbers. I prefer you can define some variables at the top of this script.